client/gutenberg/editor/main.jsx

```diff
@@ -53,13 +53,27 @@ class GutenbergEditor extends Component {
 	}
 }
 
+const addGutenbergDemoContent = post => {
+	const title = {
+		raw: 'Welcome to the Gutenberg Editor',
+		rendered: 'Welcome to the Gutenberg Editor',
+	};
+
+	return {
+		...post,
+		title,
```

Comment:
I haven't tested it, but I have a feeling that this won't work, notably at the point that we actually save. Since what we're feeding here is instructing Gutenberg to what the saved value of the post is, Gutenberg is not aware that the title has been modified and thus will not include it in the save payload. Since it won't be included in the save payload, it won't be updated server-side and thus the existing title (presumably "Auto Draft") will remain.

This is what the `overridePost` prop of the `Editor` component (and argument of `initializeEditor`) is meant to represent:

https://github.com/WordPress/gutenberg/blob/f175813c4871dd6472e33fd528bae79b6d9a24d0/edit-post/index.js#L70

This is also subject to refactoring in https://github.com/WordPress/gutenberg/pull/9403, both in form of renaming this to `initialEdits` and, in doing so, not expecting values to be passed in shape of object with `raw` / `rendered`.